src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/PInvokeModuleFixupNode.cs

```diff
@@ -37,9 +37,7 @@ public override ObjectData GetData(NodeFactory factory, bool relocsOnly = false)
             ObjectDataBuilder builder = new ObjectDataBuilder(factory);
             builder.DefinedSymbols.Add(this);
 
-            ISymbolNode nameSymbol = factory.Target.IsWindows ?
```

Comment:
Not sure if it is premature optimization but I do think there is value in consistency and simplicity, at least until we know otherwise. deleted.